Assets/VRTK/Source/Scripts/Interactions/Interactables/VRTK_InteractableObject.cs

```diff
@@ -1134,6 +1136,32 @@ protected virtual void LoadPreviousState()
             }
         }
 
+        /// <summary>
+        /// Returns the previous state of the Interactable Object
+        /// </summary>
+        /// <param name="previousParent">Out param for the previous parent</param>
+        /// <param name="previousKinematic">Out param for the previous Kinematic state</param>
+        /// <param name="previousGrabbable">Out param for the previous Grabbable State</param>
+        public virtual void GetPreviousState(out Transform previousParent, out bool previousKinematic, out bool previousGrabbable)
```

Comment:
public methods need grouping with the other public methods and appear before the protected methods